collie/model/base/trainer.py

```diff
@@ -5,7 +5,7 @@
 from pytorch_lightning import Trainer
 try:
     from pytorch_lightning.utilities.model_summary import ModelSummary
-except ImportError:
+except ImportError:  # compatible with old ``ModelSummary`` API used in versions prior to ``1.6``
```

Comment:
Is it `1.6` or `1.5`? I feel like it might be `1.5`... 

Also, I know I told you to ignore this, but now I am confusing myself. Sorry 😬

Reply:
It is 1.5!

Reply:
But the similar comment on L409 is `1.6`